cedar-policy/src/api.rs

```diff
@@ -79,8 +76,31 @@ pub mod entities {
             self.inner.size_hint()
         }
     }
+
+    /// Errors around entities
+    pub mod err {
+        pub use cedar_policy_core::entities::err::*;
```

Comment:
I'm a little wary of `pub use` with `*` from core, since it makes it easy to accidentally expose types we don't intend to. I see why it's nice not to explicitly `use` each struct though. Anyone else have thoughts?

Reply:
Yeah, I honestly think it's worth just spelling it all out. Revision on the way